test/utils.jl

```diff
@@ -35,12 +35,13 @@ local_count(oid_hex) = first(get(Ray.get_all_reference_counts(), oid_hex, 0))
 
 # Useful in running tests which require us to re-run `Ray.init` which currently can only be
 # called once as it is infeasible to reset global `Ref`'s using `isassigned` conditionals.
-macro process_eval(ex)
-    expr = QuoteNode(ex)
-    return esc(:(process_eval($expr)))
+macro process_eval(ex...)
+    kwargs = ex[1:(end - 1)]
+    expr = QuoteNode(ex[end])
+    return esc(:(process_eval($expr; $(kwargs...))))
 end
 
-function process_eval(expr::Expr; stdout=devnull)
+function process_eval(expr::Expr; stdout=devnull, stderr=devnull)
```

Comment:
```suggestion
function process_eval(expr::Expr; stdout=devnull, stderr=Pipe())
```

is this not simpler? can remove the ternary operation below then

Reply:
Good idea. The `devnull` approach was originally added as previously I'd only report logs in the error when the user didn't request any